arch/arm/boot/dts/overlays/revpi-connect4-overlay.dts

```diff
@@ -174,6 +174,12 @@
 				brcm,function = <BCM2835_FSEL_GPIO_IN>;
 				brcm,pull     = <BCM2835_PUD_OFF>;
 			};
+			id_wp_pins: id_wp_pins {
+				brcm,pins     = <17>;
+				brcm,function = <BCM2835_FSEL_GPIO_OUT>;
```

Comment:
It is a bit counter intuitive, but I would set the pin as input. Thus the external pull-up will keep the WP enabled. If it is configured as GPIO out it might drive the pin low by default and disable the WP

Reply:
But a hog gpio cannot be used by the user and therefore we wouldn't be able to toggle the wp during hat eep write?

Reply:
I was referring to gpio-hogs which afaik are requesting the gpio exclusively (like `ant1` and `ant2`). But it seems that I misunderstood you / you meant something different.

Reply:
Anyway, I've updated the last commit as you suggested